### DISCUSSION TITLE
Your model

### DISCUSSION THREAD

**Discussion by seerge** [seerge/g-helper#707] [Polls]
Hello, just wanted to collect some data about most popular models among app users :) Feel free to specify your exact model in comments.

Thanks!

**Comment by bowes332frank** [comment]
ASUS called “幻13 2022”in China Market.
I'd like Flow X13 / Z13 by the way.

**Comment by Provinomico** [comment]
I got a 2023 Zephyrus Duo16 with the R9 7945HX.

**Comment by khtpro** [comment]
hi sir, i just want to let you know that your app is so amazing than armoury crate! that i just donated a dollar into this project, i know that isn't enough but i want this app to be succeeded,, im 100% supporting this app! its also lightweight, i just only have some problem, fan control seems unsupported in my system :( .. btw im using tuf f15 2021 model and whenever i click the fan control, it block by ASUS bios, its says (asus bios rejected fan control) just hoping for an update for this if possible soon,, thank you so much for your app

**Reply by seerge** [reply]
If it says that it means your device doesn’t support fan curves

**Reply by XyKER1** [reply]
From what I've heard, the TUF lineup got fan control starting with the 2022 models, so unfortunately no application can change this (as it has to be in the BIOS).

**Reply by Draebor** [reply]
bugger about that,  got a 2021 model TUF here,  was hoping it was just a bug but yeah if bios controlled its a no go then.

**Comment by Kickskii** [comment]
Strix g15 Advantage Edition
btw any clue why platform limits are allowed to 250w on here? since armoury crate only had it till 190w? if that actually works could be potentially dangerous no?
![image](https://github.com/seerge/g-helper/assets/56576955/32e5b9b2-58cb-4621-ae86-ee642fa6e1f6)

**Reply by seerge** [reply]
Your laptop has some hardware limits, so power won’t go higher than that anyway:)

**Reply by Kickskii** [reply]
it can override PL2(slowPPT)
it makes it so laptop can run 215w(equal to pl1(fastPPT)) constantly until vrm throttle no test on higher limits cause safety

**Comment by ghost** [comment]
TUF Gaming A16 FA617NS

**Comment by ghost** [comment]
TUF Gaming A15 FX506IC

**Comment by brambedkar59** [comment]
TUF Gaming F17 2021 FX706HE

**Comment by Draebor** [comment]
TUF Gaming F17 2021 FX706HC

**Comment by sam86314** [comment]
G14 2022 GA402RJ (R9 6900HS, RX 6700S)

**Comment by DaddyK1tt3n** [comment]
Mine's an Asus ROG Strix G513QM 2021 Model - R9 5900HX RTX 3060

**Comment by kuonji-arisu** [comment]
TUF F15 (2022) FX507ZM

**Comment by chonkycarlos** [comment]
Zephyrus M15 GU502GV

**Comment by Metalogic999** [comment]
Mine's a 2023 Zephyrus G14 - Nvidia RTX 4080 GPU, AMD R9 7940HS CPU

**Comment by dmage4524** [comment]
Asus TUF Gaming A16 FA617NS

**Comment by Floating-Pin** [comment]
Asus TUF Gaming A15 FA507XI

**Comment by 404GH0ST** [comment]
Asus TUF Gaming A15 FX506IH-R565B6T

**Comment by Austinzveare** [comment]
ROG Ally.

**Comment by GlazedBelmont** [comment]
ROG Zephyrus G14 2022 GA402RJ
Wish it had the Anime Matrix but I'm glad I got it for sale a couple months ago for 929$

**Comment by d00lar** [comment]
i have tuf f15 2022 and this software is great - FINALY out of armury sh1t!!!! thanks !

**Comment by cdrkruger** [comment]
TUF Gaming A15 FA506IC

**Comment by d00lar** [comment]
i posted link to Your repo here on my YT video description/comment if You dont mind  - 
https://www.youtube.com/watch?v=da_3tlftdnY
i was fighting from a year almoust so them fix  keyboard light issues as in video, even when they fixed it 2-3 months ago - they fuc&#$ it up again in last  update and again now still it canot remember simple thing that keyboard light should be off. also after hibernation same bug is present from start of this modell till today. Your app works fine even after hibernation - shuts down light as it should ! thanks again

**Comment by lettuce8019** [comment]
Zephyrus M16 (2022 model) with i9-12900H & 3060 - **GU603ZM**

**Comment by zym234555** [comment]
Rog Strix  G733PY  with7945hx+4090

**Comment by Zephyrus-7** [comment]
TUF Gaming F15 FX506HCB

**Comment by bh108** [comment]
I have a 2022 G14. I just started using G-Helper and I like it so far.  If possible, I would like to have 2 features in G-Helper that are not in Asus Armoury Crate.  1. With Asus Armoury crate you can set when the Battery Charge limit to the battery, but it starts charging the battery when the battery level drop 3%.  I would like the ability to make it so the battery level has to drop more than more than that, maybe 20%, before the battery starts charging again.  Dell laptops allow you to control this.  2. On the G14 2022, Armoury Crate allows the main 240w barrel-port charger to continue to power the laptop after it reach the battery power limit so you can use the laptop with the power adapter plug in for long periods of time.  But if you use a 100w USB-C charger to charge the laptop, it quits does not continue to power the laptop after it reaches the power limit.  The 100w C charger turns off and then starts charging again when the battery power drops 3%.  This on and off charging of the battery is not good for the laptop so I cannot use the 100w charger as my main charger when using the laptop as a workstation.  Is there a way to make it so the 100w C charger will work the same way as the 240w barrel charger, i.e. charging with the USB-C port will work the same way as charging with the barrel port?

**Reply by seerge** [reply]
I think the way laptop charges and how it behaves on USB-C or barrel charger is defined by hardware / bios not by an app running on windows :)

**Comment by SauronSkywalker** [comment]
2023 Zephyrus Duo16 with the R9 7945HX
ROG Zephyrus Duo 16 GX650PY

**Reply by GlazedBelmont** [reply]
Oooooh, that's really cool

rip your wallet though

**Comment by Heiwa03** [comment]
2022 Zephyrus G15 GA503RS 
R9 6900HS RTX 3080

**Comment by G531GV** [comment]
Model: G531GV
I7 9750H, NVIDIA GeForce RTX 2060
Thank you for a great job.

**Comment by Raychan87** [comment]
Asus G14 (2021)
Model: GA401QM
AMD 5900HS, Nvidia Geforce RTX 3060
Very Thank you for a graat Job.

**Comment by shamoi** [comment]
Asus TUF A15 ( 2022 )
Model: FA507RR
AMD 6800HS, Nvidia Geforce RTX 3070

Keep up the good work sir !

**Comment by sirkuttin** [comment]
rog ally

**Comment by astronaut701** [comment]
Asus G15 (2021)
Mine is the GA503QR to be exact, AMD 5900HS, RTX 3070.

**Comment by AnhPham1408** [comment]
2022 Zephyrus Duo 16 GX650RW
R9 6900HX, RTX 3070Ti

**Comment by Tal-GPT** [comment]
ProArt Studiobook RTX 3070 Ti

**Comment by whyredfire** [comment]
TUF Gaming A17 (2020) FA706IU
4800H, 1660ti

**Comment by thefoxy-byte** [comment]
TUF A15 (2023) FA507NU
7735HS, 4050

**Comment by SampRession** [comment]
Asus Strix Scar III G731GU (2019)
9750H / 1660 Ti

Thanks for your app 🧡

**Comment by CSARJohn** [comment]
Zephyrus Duo 15 SE

**Comment by bendierr** [comment]
Zenbook s13 OLED (2022) UM5302  

6800U

**Comment by miguelfradinho** [comment]
TUF A15 (2023) FA507XI
7940HS / 4070 

🙏 Bless G-Helper 🙏

**Comment by jemlinus** [comment]
Zenbook Pro 14 Oled UX6404
I9-13900H / RTX 4070

**Reply by georgeslanglois** [reply]
Hey! Whats ur experience with G-Helper so far? I have a Zenbook Pro Duo and I'm curious about it

**Reply by jemlinus** [reply]
Works well so far.  I don't even notice the app so all good.

**Comment by Noisrevid** [comment]
I currently use G-Helper with two models:   2023 G14 7940HS and 2022 X13 6900HS...        G-Helper is the only reason I decided to keep ANY Asus ROG laptops to be honest..   Love your work and dedication to the project.

**Comment by oM0NG0o** [comment]
I currently use G-Helper with: 2023  G814JZ (G18) 13980HX/4080

**Comment by paslavskyit** [comment]
i'm using G-Helper on both ROG Zephyrus M15 GU502LU (2020) and ROG Strix G16 G614JV (2023)

**Comment by Ireheart** [comment]
Zephyrus G14 2020 Model, The start of the Great Zephyrus G14 line

G14 GA40IV 4900HS

**Comment by squee666** [comment]
Zephyrus Duo 2023
works well had to manually disable services because if i use the 1 click extra button it kills hotkeys for second monitor

**Comment by Kikimahe** [comment]
TUF Gaming F17 (2022) FX707ZM

**Reply by bowes332frank** [reply]
ROG FLOW X13 2022 need your help！

**Comment by StefanH1303** [comment]
UX393EA ZenBook S

**Comment by russellmm** [comment]
G14 2023 AMD 7940HS Nvidia 4060GPU

**Comment by kuonji-arisu** [comment]
Bought new laptop
TUF Gaming A17 FA707XIN
7940H 4070 Laptop

**Comment by klesajos** [comment]
I got 2021 ROG Zephyrus M16 GU603HR (with i9-11900H and RTX 3070)

**Comment by Vikman83** [comment]
I have Asus G17 (G713PV)

**Comment by SynStark** [comment]
Scar 18 13980HX/4090

**Comment by Chrystef** [comment]
Asus Rog Strix G17 (G713PI 7845HX/4070)

**Comment by iatacs19** [comment]
ProArt Studiobook 16 OLED H7604JI-DS96T

**Comment by ladytesla** [comment]
G14 GA401IV _**YR 2021**_

**Comment by tech-no-sphere** [comment]
ProArt PX13 here B)
